README.md

```diff
@@ -49,12 +49,12 @@ Add this to your `Cargo.toml`:
 c2pa = "0.11.3"
 ```
 
-If you want to read or write a manifest file, add the `file_io` dependency to your `Cargo.toml`. For example:
+If you want to read or write a manifest file, add the `file_io` feature to your `Cargo.toml`. For example:
```

Comment:
I think file_io is only needed for writing.

Reply:
You need file_io if you want to use the methods that read from a manifest file, unless you read the file on your own and pass it as. a buffer.

Reply:
Suggest simply "To read or write a manifest file, add ..."